vspec2yaml.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+#
+# (c) 2016 Jaguar Land Rover
+# (c) 2021 Robert Bosch GmbH
+# (c) 2021 Pavel Sokolov (pavel.sokolov@gmail.com)
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+#
+# Concert vspec file into flat YAML
```

Comment:
Spelling: Convert

To clarify what it does how about expanding into this:

Convert all vspec input files to a single flat YAML file.

Reply:
@crea7or: vspec2yaml is fine, after we voted in the call today. But please check the spelling as mentioned by @gunnarx. Thanks so much for the work and the change!

Reply:
@gunnarx @danielwilms done!

Reply:
Thanks @crea7or!